tests/foreman/ui/test_activationkey.py

```diff
@@ -1101,12 +1100,46 @@ def test_associate_host(self):
 
         @Assert: Hosts are successfully associated to Activation key
 
-        @Status: Manual
-
         @BZ: 1078676
 
         """
-        pass
+        key_name = gen_string("alpha", 10)
```

Comment:
please prefer `utf8` and 10 is the default length, so you are free to use `gen_string('utf8')`